fashify/src/components/Nav/Nav.jsx

```diff
@@ -58,6 +58,9 @@ export const Nav = () => {
         </div>
         <div className='nav-right'>
           <ul className='nav-links'>
+            <li className='nav-link-item nav-explore-link'>
+              <Link to={'/products'}>Explore</Link>
+            </li>
```

Comment:
even if `to` doesn't have braces and it's just a string then also it'll work  no need for braces

Reply:
I will change it Thanks